examples/amp-by-example/amp-by-example.html

```diff
@@ -1260,26 +1260,8 @@ <h3 class="mb1">
           <small>AMP by Example is a part of Accelarated Mobile Pages Project</small>
       </footer>
       <!-- End Footer -->
-        <amp-analytics type="googleanalytics" id="analytics">
-      <script type="application/json">
-        {
-          "vars": {
-            "account": "UA-73836974-1"
-          },
-          "triggers": {
-            "default pageview": {
-              "on": "visible",
-              "request": "pageview",
-              "vars": {
-                "title": "AMP by Example"
-              }
-            }
-          }
-        }
-      </script>
-    </amp-analytics>
     <amp-install-serviceworker
-      src="/sw.js"
+      src="/dist/sw.max.js"
```

Comment:
Hmm. In that case, I'll simply revert this change. I don't think `sw.js` is generated by a build, but it shouldn't affect the page for the purposes of visual diffing.